main-packages-list.ocp

```diff
@@ -12,47 +12,8 @@ ipxe-bootimgs-aarch64
 ipxe-bootimgs-x86
 ipxe-roms-qemu
 mod_ssl
-openstack-ironic >= 1:21.5.0-0.20230509150549.2983331.el9
-openstack-ironic-api >= 1:21.5.0-0.20230509150549.2983331.el9
-openstack-ironic-conductor >= 1:21.5.0-0.20230509150549.2983331.el9
-openstack-ironic-inspector >= 11.5.0-0.20230328170606.b251c1f.el9
-python3-cinderclient >= 9.1.0-0.20221128151726.730a8c7.el9
-python3-debtcollector >= 2.5.0-0.20221128140303.a6b46c5.el9
-python3-dracclient >= 8.0.0-0.20221128135758.9c7499c.el
-python3-eventlet >= 0.33.1-4.el9
-python3-gunicorn >= 20.0.4-2.el9
-python3-ironic-lib >= 5.4.1-0.20230328130611.6e7574c.el9
-python3-ironic-prometheus-exporter >= 4.2.0-0.20230308192702.e63fb2e.el9
-python3-ironicclient >= 4.9.0-0.20211209154934.6f1be06.el9
 python3-jinja2 >= 2-3.0.1-2.el9.1
-python3-keystoneauth1 >= 5.0.0-0.20221128144522.2445a5d.el9
 python3-mod_wsgi
```

Comment:
Side note: I wonder if we still need this

Reply:
fun fact: I removed the package at the beginning and httpd was complaining about that
now that I've added it back I'm getting a much worse error!
I'l try adding it back again and see if we can move forward from there

Reply:
without that package we end up with httpd not starting, complaining with `httpd: Syntax error on line 61 of /etc/httpd/conf/httpd.conf: Syntax error on line 8 of /etc/httpd/conf.modules.d/httpd-modules.conf: Cannot load modules/mod_wsgi_python3.so into server: /etc/httpd/modules/mod_wsgi_python3.so: cannot open shared object file: No such file or directory
`